experimental/GITFans/GITFans.jl

```diff
@@ -225,15 +225,15 @@ and returns `true` if the two objects are considered as equal,
 and `false` otherwise.
 The elements of the returned array are in general not sorted.
 """
-function orbit(point, generators, action, compare_func)
+function orbit(point, generators, action)
     orb = [point]
     # Note that in Julia (like in GAP),
     # the 'for' loop runs also over entries that get added
     # inside the loop.
     for b in orb
         for g in generators
             c = action(b, g)
-            if ! any(i->compare_func(i, c), orb)
+            if ! any(i-> i == c, orb)
```

Comment:
Perhaps this could also be changed?
```suggestion
            if !(c in orb)
```